lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
+                    {'url': this.options.accountSettingsPageUrl}
+                );
                 if (this.profileIsPrivate) {
-                    this._super(interpolate_text(
-                        gettext("You must specify your birth year before you can share your full profile. To specify your birth year, go to the {account_settings_page_link}"),
-                        {'account_settings_page_link': accountSettingsLink}
-                    ));
+                    this._super(
+                        HtmlUtils.interpolateHtml(
+                            gettext("You must specify your birth year before you can share your full profile. " +
+                                "To specify your birth year, go to the {account_settings_page_link}"),
+                            {'account_settings_page_link': HtmlUtils.HTML(accountSettingsLink)}
+                        )
+                    );
                 } else if (this.requiresParentalConsent) {
-                    this._super(interpolate_text(
-                        gettext('You must be over 13 to share a full profile. If you are over 13, make sure that you have specified a birth year on the {account_settings_page_link}'),
-                        {'account_settings_page_link': accountSettingsLink}
-                    ));
+                    this._super(
+                        HtmlUtils.interpolateHtml(
+                            gettext('You must be over 13 to share a full profile. If you are over 13, make sure that ' +
+                                'you have specified a birth year on the {account_settings_page_link}'),
+                            {'account_settings_page_link': HtmlUtils.HTML(accountSettingsLink)}
+                        )
+                    );
                 }
                 else {
                     this._super('');
-                }
-                return this._super();
```

Comment:
Yeah. For some reason, I think that return was overriding the other `this._super()` calls b/c it was always making the html blank instead of populating it with the correct string. Dropping that return fixed it.